falkordb_csv_loader.py

```
@@ -0,0 +1,825 @@
+
+import os
+import csv
+import argparse
+import sys
+from datetime import datetime
+from typing import Dict, List, Any
+from falkordb import FalkorDB
+
+
+class FalkorDBCSVLoader:
+    def __init__(self, host: str = "localhost", port: int = 6379, graph_name: str = "graph", csv_dir: str = "csv_output", username: str = None, password: str = None, merge_mode: bool = False, multi_graph_mode: bool = False):
```

Ruff (0.15.0)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)